templates/openstackclient/bin/tripleo-deploy.sh

```diff
@@ -4,6 +4,18 @@ umask 022
 
 export PWD=/home/cloud-admin
 CONFIG_VERSION=${CONFIG_VERSION:?"Please set CONFIG_VERSION."}
+sudo bash -c 'mkdir -p /var/run/tripleo-deploy && chown '$(whoami)' /var/run/tripleo-deploy'
```

Comment:
could also use an emptyVol for /var/run/tripleo-deploy and change the owner in the init container, but I am also ok to have it as is

Reply:
wait, if we use emptyDir for /var/run/tripleo-deploy won't that potentially get rid of the ansible log too?

Reply:
wait, if we use emptyDir for /var/run/tripleo-deploy won't that potentially get rid of the ansible log too?

Reply:
Oh, Nevermind. This is just the RUNDIR. Not workdir.

Reply:
Oh, Nevermind. This is just the RUNDIR. Not workdir.